web/styles/popovers.css

```diff
@@ -680,6 +681,11 @@ ul.popover-group-menu-member-list {
                     &.remove_subscription {
                         text-align: right;
                     }
+
+                    &.subscription_list_stream {
```

Comment:
This is what I had in mind, but is it possible to replace both `:first-of-type` and `:last-of-type` here with these classes? Those changes would make for a prep commit, and then your fixes for this specific problem would be a subsequent commit.

Reply:
Yes, we can replace :first-of-type with &.subscription_list_stream and :last-of-type with &.remove_subscription.

I have made those changes, so please take a look and let me know if there’s anything else I can improve.

Reply:
Okay. Make sure you test and put up a fresh set of screenshots, but also separate these changes into multiple commits, as I described.

Reply:
Done. Changes are split into multiple commits as requested. Tested and added fresh screenshots.